source/features/repo-age.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+import {getRepoGQL, timeAgo} from '../libs/utils';
+
+async function getRepoCreationDate(): Promise<Date> {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			createdAt
+		}
+	`);
+
+	return new Date(repository.createdAt);
+}
+
+async function init(): Promise<void> {
+	const date = await getRepoCreationDate();
+	const {interval, unit} = timeAgo(date);
+	const element = (
+		<li title={`Repository created ${date.toDateString()}`}>
+			{icons.calendar()}
+
+			<span className="num text-emphasized">{interval}</span> {unit} old
+		</li>
+	);
+
+	if (select.all('.numbers-summary li').length > 4) {
+		select('.numbers-summary li:last-child')!.before(element);
+	} else {
+		select('.numbers-summary li:last-child')!.after(element);
+	}
```

Comment:
I think here you can use `appendBefore`, it’s a helper function we wrote. It behaves like “append to number summary, but before the license field if it exists”

Reply:
Or if you're targeting the license field, perhaps it's better to look for it specifically. Other extensions might add information to this bar and could throw our count-based solution off.